cypress/support/testDate.js

```diff
@@ -0,0 +1,4 @@
+export const user = {
```

Comment:
The file name provided contains a typo. It should be 'testData.js' instead of 'testDate.js'. Please correct the file name 🙏.